src/main/java/com/labsynch/labseer/domain/StandardizationDryRunCompound.java

```diff
@@ -116,32 +125,25 @@ public static TypedQuery<Integer> findMaxRunNumber() {
 				.createQuery("SELECT max(o.runNumber) FROM StandardizationDryRunCompound o", Integer.class);
 	}
 
-	public static TypedQuery<StandardizationDryRunCompound> findStandardizationChanges() {
-		String querySQL = "SELECT o FROM StandardizationDryRunCompound o WHERE changedStructure = true OR existingDuplicateCount > 0 OR newDuplicateCount > 0 OR displayChange = true";
-		return StandardizationDryRunCompound.entityManager().createQuery(querySQL, StandardizationDryRunCompound.class);
+	public static TypedQuery<StandardizationDryRunCompound> findReadyStandardizationChanges() {
+		String querySQL = "SELECT o FROM StandardizationDryRunCompound o WHERE o.syncStatus = :syncStatus";
+		return StandardizationDryRunCompound.entityManager().createQuery(querySQL, StandardizationDryRunCompound.class).setParameter("syncStatus", SyncStatus.READY);
 	}
 
-	public static TypedQuery<Long> findParentIdsWithStandardizationChanges() {
-		String querySQL = "SELECT o.parentId FROM StandardizationDryRunCompound o WHERE changedStructure = true OR existingDuplicateCount > 0 OR newDuplicateCount > 0 OR displayChange = true";
-		return StandardizationDryRunCompound.entityManager().createQuery(querySQL, Long.class);
+	@Transactional
+	public static int getReadyStandardizationChangesCount() {
+		String querySQL = "SELECT count(o.id) FROM StandardizationDryRunCompound o WHERE o.syncStatus = :syncStatus and (o.changedStructure = true OR o.existingDuplicateCount > 0 OR o.newDuplicateCount > 0 OR o.displayChange = true)";
+		return StandardizationDryRunCompound.entityManager().createQuery(querySQL, Long.class).setParameter("syncStatus", SyncStatus.READY).getSingleResult().intValue();
```

Comment:
These queries are just updated to use our new syncStatus column instead of embedding the restandardizes logic in all of the queries.